projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -178,6 +168,28 @@ public static function permissions_callback() {
 		return current_user_can( 'manage_options' );
 	}
 
+	/**
+	 * Return true if we should initialize the My Jetpack
+	 */
+	public static function should_initialize() {
+		if ( did_action( 'my_jetpack_init' ) ) {
+			return false;
+		}
+		$should = apply_filters( 'jetpack_my_jetpack_should_initialize', true );
```

Comment:
Let's get a docblock on this filter so it can be parsed by developer.jetpack.com. There's a brief mention of this in the [coding guidelines](https://github.com/Automattic/jetpack/blob/ba2dd6c30be137baad96aafb2466d0aed7e182e5/docs/coding-guidelines.md#general) doc that points to an [example](https://github.com/Automattic/jetpack/blob/16bc2fce3ace760ff402f656dcf05255888f23f4/modules/sitemaps/sitemaps.php#L143-L151). @jeherve is this the only place we mention it in the docs?

Reply:
Addressed in 75e5f14a01f2d9c15024993ab0ca853ef2504dc3